README.md

```diff
@@ -41,6 +34,22 @@ Several other validation scripts and tools with more flexibility such as validat
 bin/validate-datapackage [-v remote_spec_ref | -l local_spec_path] [-d dataset_path]
```

Comment:
There are 5 scripts in bin/ and the usage information isn't always clear about why you'd use one script or another, in addition, the names are all very similar. I think it would help a lot to document what the purpose of each script is here.